src/webots/nodes/WbIndexedFaceSet.cpp

```diff
@@ -21,6 +21,7 @@
 #include "WbMFInt.hpp"
 #include "WbMatter.hpp"
 #include "WbNodeUtilities.hpp"
+#include "WbOdeGeomData.hpp"
 #include "WbNormal.hpp"
```

Comment:
This is not compliant with our clang format convention. (`WbNormal.hpp` and `WbOdeGeomData.hpp` should be switched).